README.md

```diff
@@ -1,3 +1,74 @@
-<h1 align="center">
-    Inside
-</h1>
+# Inside
+
+Inside is an open source plataform to deal with all the intern process of a company related to its employees. Our goal is to make easy to keep a track and manage all information related to the people that works **inside** the company.
+
+## Getting Started
+
+To get started you will need to get a copy of the project, insert the following commands (one by one) on your terminal.
+
+```javascript
+git clone git@github.com:ifactory-solutions/inside-client.git
+cd inside-client
+npm install
+```
+
+You will get a clone of the project, navigate to it, and install its depencies and then just run `npm start` to run our application over a local server.
+
+The project was initiated with the awesome facebook's cli for react (create-react-app). So you have all its default commands available, like:
+
+```
+npm run build
+npm run test
+npm start
+```
+
+### Prerequisites
+
+You should have at least the LTS node version 6.11. We recommend you to use [nvm](https://github.com/creationix/nvm) it will make easy for you to switch between node/mpm versions.
+
+
+### Installing
+
+Run `npm start` (inside you clone folder) to run our application over a local server.
```

Comment:
Run `npm start` (into the project folder) to run the application over a local server

the application isn't ours 😃